dotnet/src/webdriver/BiDi/Modules/BrowsingContext/GetTreeCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OpenQA.Selenium.BiDi.Communication;
+using System.Collections.Generic;
+
+namespace OpenQA.Selenium.BiDi.Modules.BrowsingContext;
+
+internal class GetTreeCommand(GetTreeCommandParameters @params) : Command<GetTreeCommandParameters>(@params);
+
+internal record GetTreeCommandParameters : CommandParameters
+{
+    public ulong? MaxDepth { get; set; }
```

Comment:
`ulong` is not CLS-compliant. Consumers like `VB.NET` or `F#` are affected. I guess it is not criminal just to use `long` instead of `ulong`.

Reply:
Done.